### ISSUE TITLE
Provide a way to define name to be ignored via `good-method-names` and `good-property-names`

### ISSUE THREAD

**Issue by fmigneault** [PyCQA/pylint#4563]
### Is your feature request related to a problem? Please describe

I have encountered a use case where `invalid-name` is raised, where I believe it shouldn't.
Example class: 

```python
class User():
    @property
    def id(self): 
        return # something
 ```

In that case, because `id` is 2 characters, the name is not matched against the regex rule of 3+ snake_case characters.

### Describe the solution you'd like

Similar to `good-names` that allows override of the rules employed to match valid variable names, configurations `good-method-names` and `good-property-names` should be available. 
This would allow to list very specific cases where names should be considered valid even if they disagree with the other rules. 

The reason I feel methods and properties should have their own configurations, and not mixed all together with `good-names`, is for cases similar to my `id` example. A builtin method `id()` exists. Therefore, I do not want to allow `id` variables, but the same name as property in a class seem perfectly valid and a common practice for database models in my opinion. 

I am aware that I could override the regex for 2 characters, but I would much rather have the option to allow very specific method/property names. Setting a 2 characters regex would globally allows names like `_i`, `x_`, `aa`, etc. which is not something I want to allow. 

### Additional context

relates to #2018 and #4183 that could benefit from the same configurations

**Comment by Pierre-Sassoulas** [comment]
Thank you for opening the issue, it makes sense.
